pkg/ansible/watches/watches.go

```diff
@@ -394,19 +395,25 @@ func verifyAnsiblePath(playbook string, role string) error {
 // number of workers based on their cluster resources. While the
 // author may use the CLI option to specify a suggested
 // configuration for the operator.
-func getMaxWorkers(gvk schema.GroupVersionKind, defValue int) int {
-	envVar := strings.ToUpper(strings.Replace(
+func getMaxReconciles(gvk schema.GroupVersionKind, defValue int) int {
+	envVarMaxWorker := strings.ToUpper(strings.Replace(
 		fmt.Sprintf("WORKER_%s_%s", gvk.Kind, gvk.Group),
 		".",
 		"_",
 		-1,
 	))
-	maxWorkers := getIntegerEnvWithDefault(envVar, defValue)
-	if maxWorkers <= 0 {
-		log.Info("Value %v not valid. Using default %v", maxWorkers, defValue)
+	envVarMaxReconciler := strings.ToUpper(strings.Replace(
+		fmt.Sprintf("MAX_CONCURRENT_RECONCILES_%s_%s", gvk.Kind, gvk.Group),
+		".",
+		"_",
+		-1,
+	))
```

Comment:
Nit: IMO, this would be more readable as:
```suggestion
	envVarMaxReconciler := strings.ToUpper(strings.ReplaceAll(
		fmt.Sprintf("MAX_CONCURRENT_RECONCILES_%s_%s", gvk.Kind, gvk.Group), ".", "_",
	))
```